src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -556,15 +556,15 @@ function prepareCluster(cluster, context, isExtension = false) {
           attribute.$.reportingPolicy
         )
       }
-      let storagePolicy = dbEnum.storagePolicy.any
+      let storagePolicy = dbEnum.storageOption.ram
       if (context.listsUseAttributeAccessInterface && attribute.$.entryType) {
-        storagePolicy = dbEnum.storagePolicy.attributeAccessInterface
+        storagePolicy = dbEnum.storageOption.external
```

Comment:
Is this correct for all use cases @bzbarsky-apple ?

Reply:
It is just a bug fix so the database does not have "attributeAccessInterface" as the entry (see photo) but rather "external" which is what t is supposed to be if it is in that list.
<img width="146" alt="Screenshot 2023-09-14 at 10 50 11 AM" src="https://github.com/project-chip/zap/assets/64710345/d3bd1342-821b-4321-b146-ce6d135a9930">

Reply:
I'm a little confused.  storagePolicy and storageOption are not the same thing.  Why are we storing storageOption values in storagePolicy @paulr34 ?